src/d3a/models/area/stats.py

```diff
@@ -183,18 +184,22 @@ def get_market_stats(self, market_slot_list, dso=False):
 
         return out_dict
 
-    def aggregate_exported_imported_energy(self, area):
-        if self.current_market is None:
+    def _aggregate_exported_imported_energy(self):
+        if self._area.current_market is None:
```

Comment:
```suggestion
        if self.area.current_market is None:
```

Reply:
Its only used internally in this class, so I specially defined it as private inside via `_` as prefix.

Reply:
Ah yes, Sorry, I overlooked the `_`
https://github.com/gridsingularity/d3a/pull/880/files/f3c7edb6a47cb97786138bb2e762dc1eff7d4316#r501052676